src/blocks/server.blocks/middleware/__page-title/middleware__page-title.node.js

```diff
@@ -2,30 +2,40 @@ modules.define('middleware__page-title', ['config', 'logger', 'model'], function
 
     logger = logger(module);
 
-    provide(function () {
-        return function (req, res, next) {
-            logger.debug('get title by request %s', req.url);
+    provide({
+        run: function () {
```

Comment:
I'm ok with method itself, I'm not sure about naming (it's hard to understand what it's going to do untill you real the whole source)